package.json

```diff
@@ -3,6 +3,9 @@
   "version": "1.0.0",
   "description": "This project contains a collection of pages that are meant to be used for testing various privacy and security features of browsers and browser extensions.",
   "main": "server.js",
+  "engines": {
+    "node": ">=12.19"
```

Comment:
it's either 12.19+ or 14.10+ (I know because I had to update node for this to work :) ). I hope that glitch will work with this.

Reply:
Yeah this was the minimum for `crypto.randomInt` - looks like it worked on glitch though :)